### DISCUSSION TITLE
Random port from a range

### DISCUSSION THREAD

**Discussion by iRazvan2745** [pelican-dev/panel#632] [Ideas]
a button in the nodes allocation tab from where you put the min and max port and then the amount of ports to get randomly from that min max range

**Comment by rmartinoscar** [comment]
Would be cool but i don't see how can we make an UI for that any designer ? 🙃

**Comment by notAreYouScared** [comment]
I mean... when you create allocations, you input the IP and you can input single a port, or port ranges...


I don't see this being useful...
